### PR TITLE
Update layout.html

### PR DESCRIPTION
**Pull request by kideny**
use China local CDN replace google's jquery CDN

### CODE REVIEW

File: zh/_theme/phalcon/layout.html [hunk 1]
```diff
@@ -153,7 +153,7 @@
       };
     </script>
 
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
+    <script src="http://apps.bdimg.com/libs/jquery/1.8.1/jquery.min.js"></script>
```
Comment (sergeyklay) [inline zh/_theme/phalcon/layout.html]: Could you please explain this?

Comment (kideny) [inline zh/_theme/phalcon/layout.html]: googleapis.com is very slowly in China . the new url is very fast in China. baidu.com support this cdn .  baidu is the bigest search engine like google in china .

**Comment by sergeyklay** [comment]
@SidRoberts

**Comment by kideny** [comment]
@dreamsxin Can you explain this for me ?

**Comment by sergeyklay** [comment]
@kideny No problem. Just ping to the repository maintainer

**Comment by SidRoberts** [comment]
I'll merge this in several hours. Don't worry about the merge conflict - I'll deal with it.

**Comment by SidRoberts** [comment]
Added in https://github.com/phalcon/docs/commit/b46956dcebb0720bd5799da3dcc7c0f1448e5d1b

**Pull request closed without merging**